build/apiserver/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian
+FROM centos:7
 
 ADD tmp/apiserver /opt/services/
```

Comment:
I've now added Dockerfiles for RPM installation alongside Dockerfile.dev files that copy in the binary.